ambry-api/src/main/java/com.github.ambry/clustermap/ClusterMap.java

```diff
@@ -37,6 +37,12 @@
   List<? extends PartitionId> getWritablePartitionIds();
 
   /**
+   * Gets a list of all partitions in the cluster
+   * @return a list of all partitions in the cluster
+   */
+  List<? extends PartitionId> getAllPartitions();
```

Comment:
Could this be called `getAllPartitionIds` to match the naming of the other methods in this interface